addons/project_todo/tests/test_mail_activity_todo_create.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+import datetime
+from markupsafe import Markup
+
+class TestMailActivityTodo(TestProjectCommon):
```

Comment:
Small style suggestion: surround classes with two blank lines
For reference: https://peps.python.org/pep-0008/#blank-lines